lisa/sut_orchestrator/azure/common.py

```diff
@@ -1934,9 +1934,17 @@ def copy_vhd_to_storage(
                     log=log,
                 )
             else:
-                blob_client.start_copy_from_url(
-                    src_vhd_sas_url, metadata=None, incremental_copy=False
-                )
+                try:
+                    blob_client.start_copy_from_url(
+                        src_vhd_sas_url, metadata=None, incremental_copy=False
+                    )
+                except ResourceExistsError as e:
+                    if "PendingCopyOperation" in str(e):
```

Comment:
When it happens, does the `wait_copy_blob` still work?

Reply:
I explained the scenario in the PR description. Let me know, if you need more details.

Reply:
Please add it in the code comment too. The PR is for change, and the code comment is to explain the behavior, and it exists with the code logic together.

Reply:
Added comments